README.md

```diff
@@ -1,2 +1,185 @@
+
+- [x] Displays a list of cities
+- [x] Displays weather information and forecast for a selected city.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+### 🚀 Live Demo <a name="live-demo"></a>
+- [Live Demo Link](https://weatherapp-io.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+<ul>
+  <li>GitHub Account</li>
+  <li>Git installed on your system</li>
+  <li>Install all dependencies</li>
+  <li>Code Editor</li>
+</ul>
+
+### Setup
+
+Clone this repository to your desired folder, write the following commands in git bash 
+
+```sh
+  cd my-folder
+  git clone https://github.com/davidolanrewaju/WeatherApp.git
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm init -y
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+  npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+  npx create-react-app
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+  npx eslint "**/*.{js,jsx}"
+  npm run build
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Olanrewaju David**
+- GitHub: [@davidolanrewaju](https://github.com/davidolanrewaju)
+- Twitter: [@ola_dayve](https://twitter.com/ola_dayve)
+- LinkedIn: [David Olanrewaju](https://www.linkedin.com/in/david-olanrewaju)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- VIDEO SUMMARY -->
+## 🎥 Video Summary <a name="video-summary"></a>
+>This is a video summary on the aim of this project and the challenges encountered.</br>
+Click [this link](https://www.loom.com/share/f02739603fca4d7c83ee79ed7d6c995a) to get a view of the project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/davidolanrewaju/WeatherApp/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project hit the like button and also hit me up if you need a collaborator on any side projects you're working on.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse for the resources provided to make this project possible.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Also, here. The README file is expected to contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).  Please add the appropriate Acknowledgements for [Nelson Sakwa](https://www.behance.net/sakwadesignstudio)